.eslintrc

```diff
@@ -0,0 +1,38 @@
+{
+  "extends": [
+    "airbnb-typescript",
+    "airbnb/hooks",
+    "plugin:@typescript-eslint/recommended",
+    "plugin:jest/recommended",
+    "prettier",
+    "plugin:prettier/recommended"
```

Comment:
Actually according to https://prettier.io/docs/en/integrating-with-linters.html the plugiin should not be used and just the prettier config. Since we are forcing prettier to run anyways, i'd stick with that and remove the formatting from the linting entirely

Reply:
Good point, I'll remove the prettier plugin